From 4c1d80dd2a77adf3392ca7f9479abb727636b9e5 Mon Sep 17 00:00:00 2001
From: Joe Marcus Clarke <marcus@FreeBSD.org>
Date: Sat, 23 Jan 2010 20:32:17 -0500
Subject: [PATCH 25/45] Fix the environment variable names

Now that the property names for USB interfaces have been fixed, correct
the environment variable names.
---
 hald/freebsd/probing/probe-usb2-interface.c |    4 ++--
 1 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/hald/freebsd/probing/probe-usb2-interface.c b/hald/freebsd/probing/probe-usb2-interface.c
index a41e047..9c0523a 100644
--- a/hald/freebsd/probing/probe-usb2-interface.c
+++ b/hald/freebsd/probing/probe-usb2-interface.c
@@ -58,11 +58,11 @@ main(int argc, char **argv)
   if (pbe == NULL)
     goto end;
 
-  busstr = getenv("HAL_PROP_USB_DEVICE_BUS_NUMBER");
+  busstr = getenv("HAL_PROP_USB_BUS_NUMBER");
   if (! busstr)
     goto end;
 
-  addrstr = getenv("HAL_PROP_USB_DEVICE_PORT_NUMBER");
+  addrstr = getenv("HAL_PROP_USB_PORT_NUMBER");
   if (! addrstr)
     goto end;
 
-- 
1.7.2.1

